scripts/check-devices.py

```diff
@@ -214,7 +222,7 @@ def cli_manager() -> argparse.Namespace:
     parser.add_argument(
         "-log",
         "--loglevel",
-        default="info",
+        default="unset",
```

Comment:
Can we just remove the default value and test if the value is set or not?

Reply:
Fixed with last commit